python_modules/libraries/dagstermill/dagstermill/factory.py

```diff
@@ -32,6 +32,15 @@
 from .errors import DagstermillError
 from .translator import DagsterTranslator
 
+IS_WINDOWS = os.name == "nt"
+
+
+def _clean_path_for_windows(notebook_path):
+    """In windows, the notebook cant render in dagit unless the C: prefix is removed"""
+    if IS_WINDOWS and ("c:" in notebook_path or "C:" in notebook_path):
```

Comment:
would it always be in C drive?

Reply:
confirmed that on my windows machine i can have the notebook path without the `C:` prefix and the asset executes normally. So there could be a case when a user is on windows but specified the paths without `C:` so we wouldn't want to remove the first two characters of the path

Reply:
can we use [os.path](https://docs.python.org/3/library/os.path.html) to somehow trim wins drive prefix?

Reply:
yeah that's a much better idea